src/util/json.py

```diff
@@ -16,7 +16,15 @@ def load(fp):
 else:
 
     def dumps(value, sort_keys=False):
-        option = orjson.OPT_SORT_KEYS if sort_keys else None
+        def is_namedtuple(obj):
+            return isinstance(obj, tuple) and hasattr(obj, "_fields")
+
+        if is_namedtuple(value):
```

Comment:
Can be simplified to 
```suggestion
        if isinstance(value, tuple):
            # downcast namedtuples
```